src/App.js

```diff
@@ -1,6 +1,6 @@
-import React from 'react';
```

Comment:
The changes in this files have been made by eslint so it just include cosmetics.